src/en/send.md

```diff
@@ -353,3 +341,145 @@ If the request is not successful, the response body is `json`, refer to the tabl
+
+The following lines should be your recipients' details and should match the order of column headers. You can have between 1 and 50,000 recipients.
+
+#### Optional arguments
+
+**scheduled_for (optional)**
+
+If you want to send your batch of notifications in the future, you can specify a datetime up to 4 days in the future, in [ISO 8601 format](https://en.wikipedia.org/wiki/ISO_8601). Example: `2021-06-08T15:15:00` (UTC time).
+
+**sender_id (optional)**
+
+If you want to use a specific reply-to email address when sending a batch of emails, you can specify a reply-to email address ID.
+
+To find your reply-to email address ID:
+
+1. [Sign in to GC Notify](https://notification.canada.ca/sign-in).
+1. Go to the __Settings__ page.
+1. In the __Email__ section, select __Reply-to email addresses__
+1. Copy the ID of the reply-to email address you want to use
+
+By default, GC Notify will use your default reply-to email address if you don't specify one, or none if you didn't set up one.
+
+**csv (optional)**
+
+If you prefer to pass the content of CSV files instead of rows in the `rows` argument, you can do so. Pass the full content of your CSV file in a key named `csv`. Do not include the `rows` argument.
+
+For example
+
+```json
+{
+  "name": "My bulk name",
+  "template_id": "f33517ff-2a88-4f6e-b855-c550268ce08a",
+  "csv": "email address,name\nalice@example.com,Alice"
+}
+```
+
+
+### Response
+
+::: warning Response timeout
+If you specify a response timeout when calling this endpoint, make sure it is set to 15 seconds. The GC Notify API could take a few seconds to validate your request and save the payload if you submit a large batch.
+:::
+
+If the request is successful, the response body is `json` with a status code of `201`:
+
+```json
+{
+   "data":{
+      "api_key":{
+         "id":"de1fafa2-fb2a-49c5-9b9a-8400727ecd29",
+         "key_type":"team",
+         "name":"Test"
+      },
+      "archived":false,
+      "created_at":"2021-06-10T17:14:15.341308+00:00",
+      "created_by":{
+         "id":"6af522d0-2915-4e52-83a3-3690455a5fe6",
+         "name":"Notify service user"
+      },
+      "id":"0ea216ae-4b03-46b7-ab44-893ae85104f5",
+      "job_status":"pending",
+      "notification_count":3,
+      "original_file_name":"My bulk name",
+      "processing_finished":null,
+      "processing_started":null,
+      "scheduled_for":null,
+      "sender_id":null,
+      "service":"afa2be3b-1250-430f-a70f-28a1a9d49dfa",
+      "service_name":{
+         "name":"Test service"
+      },
+      "template":"659a214f-dfec-4882-9242-fea0bd502a09",
+      "template_version":4,
+      "updated_at":null
+   }
+}
+```
+
+You can follow the progression of your batch of notifications from the GC Notify web interface.
+
+If you scheduled your batch in the future, you can cancel it from the web interface.
```

Comment:
Is there a way to do either of these things not in the interface?

Reply:
Not at the moment